mangaki/mangaki/utils/anidb.py

```diff
@@ -1,10 +1,13 @@
 from datetime import datetime
+from typing import Dict, List, Optional
 from urllib.parse import urljoin
 
 import requests
 from bs4 import BeautifulSoup
+from lazy_object_proxy.utils import cached_property
```

Comment:
My bad, it is most likely a dependence of Django which I have imported instead of `django.utils.functional`.